src/components/AgregarCliente.vue

```diff
@@ -150,7 +160,57 @@ export default {
     };
   },
   methods: {
-    ...mapActions(["agregarCliente"])
+    ...mapActions(["agregarCliente"]),
+    agregarCliente(){
+      /*
+      Variables
+      */
+     var camposVacios = true;
+     var camposNumeros = true;
```

Comment:
Esta variable ya no hace falta usarla, podés hacer todo con `camposVacios`

Reply:
uso la variable para el 2 array, para el 2 for each, sino con solo comletar los campos se puede enviar la respuesta, com facturacion negativa

Reply:
Se puede usar la misma variabe (`camposVacios`) en los dos for each, porque es un bool por lo cual no influye en los tipos de dato de los campos que están siendo evaluados. A lo sumo renombrar `camposVacios` a `camposValidos` para que quede más claro (válido puede ser que no esté vacío o que sea mayor a cero).